.github/actions/common/setup-env/action.yaml

```diff
@@ -23,5 +23,26 @@ runs:
           df -h
           sudo rm -rf /usr/share/dotnet
           sudo rm -rf /usr/local/lib/android
+
+          sudo rm -rf /opt/microsoft
+          sudo rm -rf /opt/google
+          sudo rm -rf /opt/az  # Azure
+          
+          sudo rm -rf /opt/hostedtoolcache/CodeQL
+          sudo rm -rf /opt/hostedtoolcache/Ruby
+          sudo rm -rf /opt/hostedtoolcache/go
+          sudo rm -rf /opt/hostedtoolcache/node
+
+          # sudo rm -rf /usr/lib/jvm
+
+          sudo rm -rf /usr/share/swift
+
+          sudo rm -rf /usr/local/share/powershell
+
+          # Postcommit remove dashboards
+          if [ -d dashboard ]; then
+            rm -rf dashboard
+          fi
```

Comment:
I think rm -rf already handles the case where the directory doesn't exist

Reply:
just tested it. Will remove the if condition. `rm -r <dir>` is the one that throws an error if the dir doesn't exist